features/audio.yml

```
@@ -1,9 +1,12 @@
-draft_date: 2024-06-05
 name: <audio>
-description: The `<audio>` element TODO.
+description: The `<audio>` element embeds a sound media player in a document.
```

@ddbeck the "embeds in a document" part of this strikes me as odd, as all elements add something to a document. It gives it a plugin feel, as if it's not quite integrated into the platform.

How about...

```suggestion
description: The `<audio>` element plays sound such as audio effects or music, optionally with controls provided by the browser.
```

---

Yeah, I had a time with this myself. In the end, I aligned to https://developer.mozilla.org/en-US/docs/Web/HTML/Element/audio, which wasn't _great_.

Setting aside embeds, I think the big alternative here is _represents_. As in:

```suggestion
description: The `<audio>` element represents a sound file or stream, such as audio effects or music. Optionally, the browser provides playback controls.
```

We haven't done "`<element>` represents…" with other elements so far, but we _could_. It's a bit abstract—representation doesn't _do_ anything, without further explanation—but a lot of elements _are_ abstract in this way (e.g.,`<em>` or `<div>`).

---

I'm fine with "represents" if we can point to the thing it represents. Because of MSE (`MediaSource`) and how `getUserMedia()` plugs into `<video>` with `srcObject` or a special URL, media elements don't necessarily represent files. You could say they represent media resources, but that's quite abstract.